colour/appearance/ciecam02.py

```diff
@@ -1344,7 +1346,8 @@ def P(N_c, N_cb, e_t, t, A, N_bb):
     A = np.asarray(A)
     N_bb = np.asarray(N_bb)
 
-    P_1 = ((50000 / 13) * N_c * N_cb * e_t) / t
+    with np.errstate(divide='ignore'):
```

Comment:
Any reason to specifically ignore the division error here?

We tend to not mute any warnings by default however there is a global switch in the API to mute them if desired: [colour.filter_warnings](https://github.com/colour-science/colour/blob/develop/colour/utilities/verbose.py#L144)

and we have a bunch of decorators here too: https://github.com/colour-science/colour/blob/develop/colour/utilities/common.py

Everything else looks good!